src/closure-externs/closure-externs.js

```diff
@@ -268,3 +268,6 @@ Navigator.prototype.webkitGetUserMedia = function(
  * @type {symbol}
  */
 Symbol.dispose;
+
+// Common between node-externs and v8-externs
+var os = {};
```

Comment:
Maybe we could have instead added this to v8.js but with `@suppress {duplicate}` like some(most) of the other stuff in that file?

Reply:
I can open a PR doing that